src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -199,8 +199,14 @@ function Base.show(io::IO, ::MIME"text/plain", limiter::SubcellLimiterIDP)
                 ]
             end
             if positivity
-                string = "Positivity limiting for conservative variables $(limiter.positivity_variables_cons) and $(limiter.positivity_variables_nonlinear)"
-                setup = [setup..., "" => string]
+                if !isempty(limiter.positivity_variables_cons)
+                    string = "conservative variables $(limiter.positivity_variables_cons)"
+                    setup = [setup..., "" => "Positivity limiting for " * string]
+                end
+                if !isempty(limiter.positivity_variables_nonlinear)
+                    string = "$(limiter.positivity_variables_nonlinear)"
+                    setup = [setup..., "" => "Positivity limiting for " * string]
+                end
```

Comment:
Was this causing extra output for `TreeMesh` and `StructuredMesh`?

Reply:
It's the same for all meshes. I just noticed here that if e.g. there is no positivity limiting of conservative variables, the output was like `Positivity limiting for [] and [pressure].`.